packages/data/src/registry.js

```diff
@@ -322,9 +322,14 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 
 		emitter.pause();
 		Object.values( stores ).forEach( ( store ) => store.emitter.pause() );
-		callback();
-		emitter.resume();
-		Object.values( stores ).forEach( ( store ) => store.emitter.resume() );
+		try {
```

Comment:
Should we also wrap with `try/catch/finally` the `callback()` that we run if we're already batching above (L319)?

Reply:
L319 returns early and doesn't touch emitters. Bubbling the errors should be okay there.